src/schema/work.js

```diff
@@ -70,14 +70,23 @@ export const resolvers = {
     },
 
     async localizations(parent, args, context, info) {
-      // collect pids from manifestations
-      const pids = parent.manifestations.map(
-        (manifestation) => manifestation.id
+      // @TODO - check hostpublicationpid ..
+      // @TODO should we do this for articles only ??
+      // @TODO or acutally we should not do this at all but get it from workservice
+      const allmanifestaions = await Promise.all(
+        parent.manifestations.map((manifestation) => {
+          return context.datasources.openformat.load(manifestation.id);
+        })
+      );
+
+      const pids = allmanifestaions.map((manifestation) =>
+        manifestation?.details?.hostPublicationPid?.$
+          ? manifestation?.details?.hostPublicationPid?.$
+          : manifestation.admindata.pid.$
```

Comment:
Vil denne sti altid eksistere på alle manifestationer? `manifestation.admindata.pid.$`

ellers: 
```suggestion
          const pids = allmanifestaions.map((manifestation) =>
        manifestation?.details?.hostPublicationPid?.$ || manifestation?.admindata?.pid?.$
```